deploy/pipeline.yml

```diff
@@ -155,6 +155,7 @@ Resources:
               - 's3:GetObject'
               - 's3:GetObjectVersion'
               - 's3:GetBucketVersioning'
+              - 'ssm:*'
```

Comment:
I would suggest you scope this a bit more: are you just needing the ability to read parameters, and if so just on a certain path? You can take a look at https://github.com/ndlib/remix-blueprints/blob/master/deploy/cloudformation/static-host-pipeline.yml#L117-L141 for an example of allowing a full path to be read.